### DISCUSSION TITLE
opensumi的集群部署实践架构

### DISCUSSION THREAD

**Discussion by ainflying** [opensumi/core#1784] [Ideas]
就是我们的opensumi部署在单机上，单机挂了，数据就没有了。怎么像其它应用那样支持集群方式的部署？

**Comment by hacke2** [comment]
企业内部需要自建 Kubernetes 集群，并且有镜像服务可以存储镜像，存储相关服务（提供storageclass）：rbd-class，cephfs-class以及nas盘均可满足，性能越好的存储越符合需求。
这样即便容器挂了，代码依旧再的，做的再细一点需要将盘的内容备份到 OSS 上，这个盘也就可以销毁了

**Reply by minalwws** [reply]
想知道有没有 集群部署实践 的教程 和 指导，谢谢
